keps/sig-api-machinery/20190228-priority-and-fairness.md

```diff
@@ -775,129 +788,188 @@ Some flow schemata.
 ```yaml
 kind: FlowSchema
 meta:
-  name: system-top
+  name: exempt
 spec:
-  requestPriority:
-    name: system-top
-  match:
-  - and: # writes by admins (does this cover loopback too?)
-    - superSet:
-      field: groups
-      set: [ "system:masters" ]
+  matchingPrecedence: 1500
+  priorityLevelConfiguration:
+    name: exempt
+  rules:
+  - rule: # All resource requests from system-privileged group
+      verbs: ['*']
+      apiGroups: ['*']
+      resources: ['*']
+    subjects:
+    - kind: Group
+      name: "system:masters"
+  - rule: # Delegated auth[n/z] requests
```

Comment:
I think the matching condition here could be tightened up.  Have a look at the example data at https://github.com/kubernetes/kubernetes/pull/81224#issuecomment-521835562

Reply:
no, ` ['authentication.k8s.io', 'authorization.k8s.io']` is sufficient

Reply:
Do you mean that no other requests involve those API groups?  I doubt it.  I am suggesting that we be more discriminating here, giving this exceptionally high priority only to requests that really need it, as best we can distinguish.

Reply:
yes, the group is dedicated for delegated authn/authz

Reply:
Oh, I see.  OK then.  We could still tighten up a bit, by requiring that the user be a service account --- but that still leaves a lot of access to this high priority.